src/wrappers/foresight/lib/jest/ModuleLoader.ts

```diff
@@ -0,0 +1,54 @@
+import ModuleUtils from '../../../../utils/ModuleUtils';
+import {
+    INTEGRATIONS,
+    WRAPPERS,
+} from '../../../../Constants';
+import ThundraLogger from '../../../../ThundraLogger';
+
+/**
+ * try to load test modules
+ * @param testRequire testRequire testsuite's context require
+ */
+const LoadTestModules = (testRequire: any) => {
+
+    loadIntegrations(testRequire);
+    loadWrappers(testRequire);
+};
+
+const loadIntegrations = (testRequire: any) => {
+
+    for (const key in INTEGRATIONS) {
+        const integration = INTEGRATIONS[key];
+        if (integration && !integration.buildInIntegration) {
```

Comment:
I think, `buildInIntegration` configuration and its check here should not be needed as such modules (http, https) are already ignored by ModuleUtils if they are retried to be instrumented